docs/dm/zh_CN/features/table-route.md

```diff
@@ -0,0 +1,34 @@
+table route rule
+===
+
+#### 功能介绍
+
+table route 提供将上游 MySQL/MariaDB 实例的某些表同步到下游指定表的功能。
```

Comment:
emphasize the upstream name and downstream name can be different?

Reply:
let user check